### DISCUSSION TITLE
Do we need/want colony IDs?

### DISCUSSION THREAD

**Discussion by janaobsteter** [HighlanderLab/SIMplyBee#283] [Q&A] [answered]
We were discussing about having unique colony IDs. It has proven in many situations that this would indeed be useful. What do you think? And also, if we have them, what should they be?

**Comment by gregorgorjanc** [comment]
I think we should have them in the SimParamBee and they would be simply 1:n. Every new colony would get +1. Collapsed colony ID would never get reused.

**Reply by gregorgorjanc** [reply]
@janaobsteter can you mark this as answered?

**Comment by janaobsteter** [comment, marked as answer]
This is now implemented in SIMplyBee
